hashed-io/hashed-substrate
# Update version of pokadot and several changes for fund admin

Updates to polkadot v0.9.33
Adds a template to run the migrations on the template pallet Fixes some issues with found admin pallet
Add storage version on all pallets

* Add schedule pallet and add storage version on the template

* remove schedualer

* Update to polkadot polkadot-v0.9.28

* Update cargo lock and remove old function un fruniques

* update cargo lock

* update cargo lock

* WIP update to polkadot v0.9.28

* Bumping versions - v0.9.33

* wip missingn to update chain spects

* update chain specs

* Bumping versions - v0.9.33 - Fixes

* [v0.9.33] Fix BlockWeights and removig #[transactional]

* switch runtime version

* update migrations for the template

* Update mock

* Update migrations and fix tests for marketplace

* update pallet to latest version

* fix types

* fix digest items

* udpate mock

* Update check action

* Add weights template and fix github actions

* Update extrinsic to latest version

* Start adding benchmarks to pallet tempalte add header for licence
adds a script to run the benchmarks
updates the runtime

* Add currency call and update benchmark script

* comment unused line

* Add storage versions to all pallets

* Remove duplicate extrincit command

* Fix bad validation

* update chain spec fix validation in fund admin

* Update validation for reset drawdowns in eb5

Co-authored-by: Artur Gontijo <arturgontijo@gmail.com>
